packages/launchpad-navigator/src/icon/index.tsx

```diff
@@ -13,9 +17,13 @@ const LaunchpadNavigatorIcon = ( { siteSlug }: LaunchpadNavigatorIconProps ) =>
 	const currentNavigatorChecklistSlug =
 		select( LaunchpadNavigator.store ).getActiveChecklistSlug() || null;
 
+	const launchpadOptions = {
+		onSuccess: sortLaunchpadTasksByCompletionStatus,
+	};
+
 	const {
 		data: { checklist },
-	} = useLaunchpad( siteSlug, currentNavigatorChecklistSlug );
+	} = useLaunchpad( siteSlug, currentNavigatorChecklistSlug, launchpadOptions );
```

Comment:
Good catch! This makes me think back to a comment I made earlier -- it feels like creating a `useSortedLaunchpadTasks` hook may be well worth it.